src/components/PointsButton.tsx

```diff
@@ -19,12 +19,5 @@ export default function PointsButton({ style, size, testID }: Props) {
     navigate(Screens.PointsHome)
   }
 
-  return (
-    <TopBarIconButton
-      testID={testID}
-      icon={<AttentionIcon size={size} />}
-      onPress={onPress}
-      style={style}
-    />
-  )
+  return <TopBarIconButtonV2 icon={<AttentionIcon size={24} />} testID={testID} onPress={onPress} />
```

Comment:
Same questions as above about `style` and using the `size` prop rather than hard-coding 24.